precompiles/gov/IGov.sol

```diff
@@ -106,4 +122,33 @@ interface IGov {
         external
         view
         returns (WeightedVote[] memory votes, PageResponse memory pageResponse);
+
+    /// @dev getDeposit returns the deposit of a single depositor for a given proposalId.
+    /// @param proposalId The proposal id
+    /// @param depositor The address of the depositor
+    /// @return deposit The deposit information
+    function getDeposit(
+        uint64 proposalId,
+        address depositor
+    ) external view returns (DepositData memory deposit);
```

Comment:
Can't we return just the Coin of DepositData instead of info provided already as input?
```suggestion
    ) external view returns (DepositData memory deposit);
```

Reply:
Let's keep the cosmos-sdk interface for now

Reply:
We could but I prefer sticking 1 to 1 with the return type from the SDK